meeting-notes/2019/Q1/2019-02-04--go-core-dev-team-weekly.md

```diff
@@ -0,0 +1,138 @@
+
+Please note. There has been a  [meeting format change](https://github.com/ipfs/team-mgmt/issues/827#issuecomment-452157617) intended to make our brief sync time more valuable and more easily understood by new/occasional attendees.
+
+**Meeting Lead:** @eingenito
+
+**Attendees:**
+  - @alanshaw
+  - @Kubuxu
+  - @mgoelzer
+  - @aschmahmann
+  - @hannahhoward
+  - @bigs
+  - @michaelavila
+  - @stebalien
+  - @mikeal
+  - @magik6k
+  - @momack2
+  - @raulk
+  
+**Recording:** 
+[This recording is private to the Go Core Dev Team](https://drive.google.com/drive/u/2/folders/1jnVNjjtNmYz0soHyg6IqiIL_EGgLXY5f)
+
+
+## Agenda
+- Review Announcements for the whole team
+- Review blockers and needs from other team members (not general updates)
+  - Are you blocked waiting for input?
+  - PRs for review? Remember the [project board](https://github.com/orgs/ipfs/projects/1)
+- Review progress on major initiatives
+  - Find or create the section for the IPFS initiative(s) you worked on.
+  - Summarize the initiative briefly in notes to give a little context.
+  - Summarize your progress
+  - Summarize your next steps.
+- Review other Questions/Announcements/Notes
+
+## Announcements
+
+- Welcome back KUBA!
+
+## Current IPFS Initiatives
+#### Datastore
+Summary: We are investigating various IPFS datastore options with the goal of replacing the current default datastore with a faster, more scalable one.
+
+- need to update the epic with the outstanding work
+
+#### Bitswap
+Summary: We are improving the performance of transferring data between IPFS instances: [Epic Issue](https://github.com/ipfs/go-ipfs/issues/5723)
+
+- a few outstanding PRs on the refactor stuff
+- a few sessions improvements, but not seeing huge improvements in real world testing
+- after refactors there will only be bitswap sessions (no get without session)
+- @hannahhoward heavily committed to GraphSync
+
+#### CoreAPI
+Summary: We are improving the API used to drive IPFS functionality and extracting it to a separate project without a dependency on go-ipfs itself. [Epic Issue](https://github.com/ipfs/go-ipfs/issues/4498)
+
+@magik6k
+  - ipfs get uses CoreAPI!
+  - ipfs ls uses CoreAPI too!
+  - go-unixfs seek fix merged (it also uses ipld walker now)
+  - block put --pin needed for http dag service - https://github.com/ipfs/go-ipfs/pull/5969
+
+#### Provider Records
+Summary: We are improving the scalability of the providing subsystem (the system for advertising, via the DHT, the content that a given IPFS node can provide). [Epic Issue](https://github.com/ipfs/go-ipfs/pull/5870)
+
+- Issue is up-to-date (https://github.com/ipfs/go-ipfs/pull/5870)
+- Currently reworking reprovider onto new provider system
+
+#### Base32/CIDv1
+
+Summary: We are converting the default format of content identifiers in IPFS from Base58 to Base32 and encoding additional metadata about the identifer. [Epic Issue](https://github.com/ipfs/go-ipfs/issues/5358)
+
+@stebalien - updates needed to providers
```

Comment:
@Stebalien would you be able to output a timeline for this feature?

@alanshaw mind making a PR out of https://github.com/ipfs/interop/tree/test/cid-version-agnostic/test so that the go-ipfs team can use those tests as well?

Reply:
like this? https://github.com/ipfs/interop/pull/46